cli/host.go

```diff
@@ -161,28 +162,41 @@ func (cmd *HostTerminateCommand) Execute(_ []string) error {
 // HostSetupCommand runs setup.sh to set up a host.
 type HostSetupCommand struct {
 	WorkingDirectory string `long:"working_directory" default:"" description:"working directory"`
+	ScriptDirectory  string `long:"script_directory" default:"" description:"directory of setup script"`
 	SetupAsSudo      bool   `long:"setup_as_sudo" description:"run setup script as sudo"`
 }
 
 // Execute runs a script called "setup.sh" in the host's working directory.
 func (c *HostSetupCommand) Execute(_ []string) error {
-	out, err := c.runSetupScript(context.TODO())
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	if c.ScriptDirectory == "" {
+		usr, err := user.Current()
```

Comment:
On windows this finds the windows home, not the cygwin home. It might be better to default to nothing so that the agent uses the home directory, since that's where it starts.